### PR TITLE
Add first version of Dockerfile for Octave

### PR DESCRIPTION
**Pull request by sbesson**
This work is derived from https://github.com/schickling/dockerfiles/tree/master/octave

/cc @bramalingam @joshmoore

### CODE REVIEW

File: bfoctave/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Octave
+
+Lightweight development environment for testing the Bio-Formats Octave package
+
+## Usage
+
+```
+$ docker run --rm -it -v $(pwd):/source ome/bfoctave
```
Comment (joshmoore) [inline bfoctave/README.md]: Our org is "openmicroscopy"

**Comment by sbesson** [comment]
Pushed. I am also reconsidering  `openmicroscopy/octave` instead of `openmicroscopy/bfoctave` assuming the goal is to have an image with Octave only. It then becomes the testers responsibility to install the Bio-Formats Octave package in the container.

**Comment by joshmoore** [comment]
That's fair. We can always add a bfoctave which extends octact and simply
downloads latest/bfoctave.zip

**Comment by sbesson** [comment]
Pushed.

**Comment by sbesson** [comment]
Closed in favor of https://github.com/sbesson/octave-docker

**Pull request closed without merging**